packages/interfaces/src/libp2p.ts

```diff
@@ -22,11 +22,3 @@ export type Libp2pComponents = Parameters<
 export type Libp2p = BaseLibp2p<Libp2pServices> & {
   components: Libp2pComponents;
 };
-
-export type CreateLibp2pOptions = Libp2pOptions & {
```

Comment:
nit: not sure if this change is intended to be part of this PR

Reply:
if it is, a separate commit would be better